tools/ci/run-integration-test.sh

```diff
@@ -29,18 +29,18 @@ cd "${WORKSPACE}"
 msg "installing dependencies"
 # NOTE: github action ros-tooling/setup-ros@0.2.1 should be run before
 sudo apt install -y apt-transport-https
-sudo apt update && sudo apt install -y cmake libacl1-dev libncurses5-dev pkg-config ros-foxy-ros-testing
+sudo apt update && sudo apt install -y cmake libacl1-dev libncurses5-dev pkg-config ros-iron-ros-testing
 
 msg "sourcing ROS workspace"
 # shellcheck source=/dev/null
-source /opt/ros/foxy/setup.bash
-
-msg "checking copyrights"
-sudo rm -rf /opt/ros/foxy/lib/python3.8/site-packages/ament_copyright/template/apache2_header.txt
-sudo cp -rf tools/apache2_header.txt /opt/ros/foxy/lib/python3.8/site-packages/ament_copyright/template/.
-# shellcheck disable=SC2026
-sudo sed -i '41 c\"'c'", "'cc'", "'cpp'", "'cxx'", "'h'", "'hh'", "'hpp'", "'hxx'", "'inl'", "'sh'"' /opt/ros/foxy/lib/python3.8/site-packages/ament_copyright/main.py
-ament_copyright --exclude LICENSE CONTRIBUTING.md tools/apache2_header.txt
+source /opt/ros/iron/setup.bash
+
+#msg "checking copyrights"
+#sudo rm -rf /opt/ros/iron/lib/python3.8/site-packages/ament_copyright/template/apache2_header.txt
+#sudo cp -rf tools/apache2_header.txt /opt/ros/iron/lib/python3.8/site-packages/ament_copyright/template/.
+## shellcheck disable=SC2026
+#sudo sed -i '41 c\"'c'", "'cc'", "'cpp'", "'cxx'", "'h'", "'hh'", "'hpp'", "'hxx'", "'inl'", "'sh'"' /opt/ros/iron/lib/python3.8/site-packages/ament_copyright/main.py
+#ament_copyright --exclude LICENSE CONTRIBUTING.md tools/apache2_header.txt
```

Comment:
```suggestion
msg "checking copyrights"
sudo rm -rf /opt/ros/iron/lib/python3.10/site-packages/ament_copyright/template/apache2_header.txt
sudo cp -rf tools/apache2_header.txt /opt/ros/iron/lib/python3.10/site-packages/ament_copyright/template/.
# shellcheck disable=SC2026
sudo sed -i '41 c\"'c'", "'cc'", "'cpp'", "'cxx'", "'h'", "'hh'", "'hpp'", "'hxx'", "'inl'", "'sh'"' /opt/ros/iron/lib/python3.10/site-packages/ament_copyright/main.py
ament_copyright --exclude LICENSE CONTRIBUTING.md tools/apache2_header.txt
```

Reply:
Scratch that, it is better to be based on ROS 2 Humble here which is the LTS version.